src/components/Dailog.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div class="main_container">
+    <div class="container" v-for="issue in issues" :key="issue.id">
+      <v-alert v-show="showAlert(issue)" class="dailog" type="error"
+        >{{ issue.bName }} to be returned on {{ issue.dueDate }}
+      </v-alert>
+    </div>
+  </div>
+</template>
+
+<script>
+import axios from "axios";
+export default {
+  name: "alertDailog",
+  data() {
+    return {
+      user: {},
+      issues: [],
+    };
+  },
+  methods: {
+    async loadData() {
+      let user = localStorage.getItem("user-info");
+      let res = await axios.get("http://localhost:3000/issues");
+      this.user = JSON.parse(user);
+      this.issues = res.data.filter((issue) => {
+        return issue.uName === this.user.email;
+      });
+    },
+    showAlert(issue) {
```

Comment:
what alert does this show? can you name it accordingly? eg: if it shows due books alert then name it showDueBooksAlert().